introduction_to_applying_machine_learning/video_game_sales/video-game-sales-xgboost.ipynb

```diff
@@ -93,15 +75,13 @@
     "---\n",
     "## Data\n",
     "\n",
-    "Before proceeding further, you'll need to sign in to Kaggle or create a Kaggle account if you don't have one.  Then **upload the raw CSV data set from the above Kaggle link to the S3 bucket and prefix you specified above**.  The raw_data_filename specified below is the name of the data file from Kaggle, but you should alter it if the name changes. Let's download the data from your S3 bucket to your notebook instance, where it will appear in the same directory as this notebook.  Then we'll take an initial look at the data."
+    "Before proceeding further, you'll need to sign in to Kaggle or create a Kaggle account if you don't have one. Then **upload the raw CSV data set from the above Kaggle link to the S3 bucket and prefix you specified above**. The raw_data_filename specified below is the name of the data file from Kaggle, but you should alter it if the name changes. Let's download the data from your S3 bucket to your notebook instance, where it will appear in the same directory as this notebook. Then we'll take an initial look at the data."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "raw_data_filename = 'Video_Games_Sales_as_at_22_Dec_2016.csv'\n",
```

Comment:
I realized this isn't being changed in this PR, but the `s3.Bucket()` call should specify the region name to ensure that it works in all regions, including GovCloud

Reply:
Will change.

Reply:
sorry, I should've been clearer with what I meant here. Instead of 

```python
s3 = boto3.resource('s3')
```

it should be:

```python
region = sagemaker.Session().boto_region_name
s3 = boto3.resource('s3', region_name=region)
```

`region` could be defined in the setup code cell, and since this is the third instance of instantiating `sagemaker.Session()`, you may want to just do `sagemaker_session = sagemaker.Session()` in that first cell as well.

Reply:
I have made these changes, will push them up.